### PR TITLE
Add cohort data view for participants

### PR DESCRIPTION
**Pull request by ktbee**
This PR adds a scenarios data view for a participant when they visit "My Cohorts"

Screen Shot 2019-12-10 at 6 03 48 PM

### CODE REVIEW

File: client/components/Facilitator/Components/Cohorts/Cohort.jsx [hunk 1]
```diff
@@ -137,12 +137,17 @@ export class Cohort extends React.Component {
...
+            cohort: { id, name },
+            user: { username, permissions }
         } = this.props;
         const { activeTabKey, tabs } = this.state;
         const { onClick, onTabClick, onDataTableClick } = this;
         const cohortUrl = `${location.origin}/cohort/${cohort.id}`;
         const source = tabs.find(tab => tab.menuItem.key === activeTabKey);
+        const currentUserInCohort = cohort.users.find(cohortMember => {
+            return cohortMember.username === username;
+        });
```
Comment (ktbee) [inline client/components/Facilitator/Components/Cohorts/Cohort.jsx]: Get the cohort data for the current user

File: client/components/Facilitator/Components/Cohorts/Cohort.jsx [hunk 2]
```diff
@@ -232,6 +237,21 @@ export class Cohort extends React.Component {
                                 onClick={onClick}
                             />
                         </ConfirmAuth>
+                        {currentUserInCohort && (
```
Comment (ktbee) [inline client/components/Facilitator/Components/Cohorts/Cohort.jsx]: currentUserInCohort is undefined initially, so check for it before trying to display this component.

File: client/components/Facilitator/Components/Cohorts/CohortDataTableMenu.jsx [hunk 3]
```diff
@@ -34,14 +35,16 @@ export class CohortDataTableMenu extends React.Component {
...
+                <ConfirmAuth requiredPermission="edit_scenarios_in_cohort">
+                    <Popup
+                        content="Close this data table tab"
+                        trigger={
+                            <Menu.Item name="close" onClick={onClick}>
+                                <Icon name="close" />
+                            </Menu.Item>
+                        }
+                    />
+                </ConfirmAuth>
```
Comment (ktbee) [inline client/components/Facilitator/Components/Cohorts/CohortDataTableMenu.jsx]: The close button doesn't make a lot of sense in the context of the participant data view

**Comment by rwaldron** [comment]
This looks great! I'll review it as soon as I can

File: client/components/Facilitator/Components/Cohorts/CohortDataTableMenu.jsx (hunk 3 above)
Comment (rwaldron) [reply client/components/Facilitator/Components/Cohorts/CohortDataTableMenu.jsx]: Brilliant

File: client/components/Facilitator/Components/Cohorts/Cohort.jsx [hunk 4]
```diff
@@ -232,6 +237,21 @@ export class Cohort extends React.Component {
...
+                            >
+                                <CohortDataTable
+                                    source={{
+                                        cohortId: id,
+                                        participantId: currentUserInCohort.id
+                                    }}
+                                    onClick={onDataTableClick}
+                                />
+                            </ConfirmAuth>
+                        )}
```
Comment (rwaldron) [inline client/components/Facilitator/Components/Cohorts/Cohort.jsx]: Solid

**Pull request merged**